src/fights.ts

```diff
@@ -573,17 +574,25 @@ class FreeRunFight extends FreeFight {
   }
 }
 
-const pygmyMacro = Macro.if_(
-  $monster`pygmy bowler`,
-  Macro.trySkill($skill`Snokebomb`).item($item`Louder Than Bomb`)
-)
-  .if_(
-    $monster`pygmy orderlies`,
-    Macro.trySkill($skill`Feel Hatred`).item($item`divine champagne popper`)
-  )
-  .if_($monster`pygmy janitor`, Macro.item($item`tennis ball`))
-  .if_($monster`time-spinner prank`, Macro.basicCombat())
-  .abort();
+const pygmyMacro = () =>
+  Macro.if_($monster`pygmy bowler`, Macro.trySkill($skill`Snokebomb`).item($item`Louder Than Bomb`))
+    .if_(
+      $monster`pygmy orderlies`,
+      Macro.trySkill($skill`Feel Hatred`).item($item`divine champagne popper`)
+    )
+    .if_($monster`pygmy janitor`, Macro.item($item`tennis ball`))
+    .if_($monster`time-spinner prank`, Macro.basicCombat())
+    .externalIf(
```

Comment:
I don't understand the reasoning of this additional combat logic, but wanted to make sure it should be included in this change.

Reply:
I didn't touch the contexts of janitors, bowlers, or orderlies, or even time-pranks--i changed it to a macro-generating function and added the part referring to drunks specificlally